client/webserver/site/src/js/app.ts

```diff
@@ -887,9 +888,13 @@ export default class Application {
   async signOut () {
     const res = await postJSON('/api/logout')
     if (!this.checkResponse(res)) {
-      // TODO: Esp. for active orders error, let the user know what's going on.
-      // Maybe give them an option to force.
-      Doc.hide(this.page.profileBox)
+      const activeOrdersErrCode = 35
```

Comment:
We have this defined at https://github.com/decred/dcrdex/blob/34732a8b7e52346d86e52a03610845e450125059/client/webserver/site/src/js/wallets.ts#L37

Seems like we should have all http API error codes that we need defined somewhere central.  We had a constants.js at one point, but that got nuked.  Thoughts?  Maybe in http.ts or at the top of app.ts?

Reply:
I think that's a good idea. I put them in http.ts.